### ISSUE TITLE
include native prometheus exporter to the image

### ISSUE THREAD

**Issue by jouve** [bitnami/bitnami-docker-haproxy#3]
### Name and Version

bitnami/haproxy:latest

### What is the problem this feature will solve?

the feature allows to expose prometheus metrics without an additional exporter

### What is the feature you are proposing to solve the problem?

Starting in version 2.0, you can compile HAProxy with native Prometheus support and expose a built-in Prometheus endpoint.

https://www.haproxy.com/blog/haproxy-exposes-a-prometheus-metrics-endpoint/

### What alternatives have you considered?

https://github.com/prometheus/haproxy_exporter

**Comment by javsalgar** [comment]
Hi! Thank you so much for the input. I will create a task for evaluation. I cannot guarantee an ETA but as soon as there are news I will update this ticket.

**Comment by alemorcuq** [comment]
We released a new version of the container (`2.6.0-debian-10-r2`) with Prometheus support:

```
$ docker run --rm -ti bitnami/haproxy:2.6.0-debian-10-r2 -vv | grep Prometheus
Built with the Prometheus exporter as a service
```
